dummyserver/server.py

```diff
@@ -267,12 +272,16 @@ def get_unreachable_address():
 if __name__ == '__main__':
     # For debugging dummyserver itself - python -m dummyserver.server
     from .testcase import TestingApp
-    host = '127.0.0.1'
+    # host = '127.0.0.1'
+    host = '/tmp/dummyserver.sock'
 
     io_loop = tornado.ioloop.IOLoop.current()
     app = tornado.web.Application([(r".*", TestingApp)])
     server, port = run_tornado_app(app, io_loop, None,
                                    'http', host)
     server_thread = run_loop_in_thread(io_loop)
 
-    print("Listening on http://{host}:{port}".format(host=host, port=port))
+    if port:
+        print("Listening on http://{host}:{port}".format(host=host, port=port))
+    else:
+        print("Listening on unix@{host}".format(host=host))
```

Comment:
Let's change this scheme for this message to be the same as what we're using as our scheme below (ie `http+unix`)